### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by ouziel-slama**
- [x] Remove extra zeros in logging
- [x] Tweak logging
- [x] Don't start the fetcher just to stop it
- [x] Tweak and fixes
----

* [x] Double-check the spelling and grammar of all strings, code comments, etc.
* [x] Double-check that all code is deterministic that needs to be
* [x] Add tests to cover any new or revised logic
* [x] Ensure that the test suite passes
* [x] Update the project [release notes](release-notes/)
* [x] Update the project documentation, as appropriate, with a corresponding Pull Request in the [Documentation repository](https://github.com/CounterpartyXCP/Documentation)

### CODE REVIEW

**Review by warrenpuffett — APPROVED** [review]

File: counterparty-rs/src/indexer/logging.rs [hunk 1]
```diff
@@ -38,7 +38,7 @@ pub fn setup_logging(config: &Config) {
 
         let stderr_layer = layer()
             .event_format(CustomFormatter {
-                timer: ChronoLocal::rfc_3339(),
+                timer: ChronoLocal::new("%Y-%m-%dT%H:%M:%S%.6f%:z".to_string()),
```
Comment (adamkrellenstein) [inline counterparty-rs/src/indexer/logging.rs:41]: I actually think millisecond precision everywhere is enough...

File: counterparty-core/counterpartycore/server.py [hunk 2]
```diff
@@ -733,6 +733,9 @@ def start_all(args):
     except KeyboardInterrupt:
         logger.warning("Keyboard interrupt!")
         pass
+    except Exception as e:
```
Comment (adamkrellenstein) [inline counterparty-core/counterpartycore/server.py:736]: This is to address this problem:

```
2024-07-05T08:03:48.967441000+00:00 - [   DEBUG] - Block 2866232 fetched. (1.067364s)
2024-07-05T08:03:48.968364000+00:00 - [    INFO] - Block 2866232
2024-07-05T08:03:48.974233000+00:00 - [   DEBUG] - Block 2866232 - Initialising dispensable cache...
2024-07-05T08:03:48.995795000+00:00 - [    INFO] - Block 2866232 - Parsing complete. L: 20557d2, TX: 65627be, M: 5d14267 (0.03s)
2024-07-05T08:03:48.997118000+00:00 - [   DEBUG] - Block 2866232/2866232 parsed, for 1 blocks in 0h 0m 1s.
2024-07-05T08:03:49.466955000+00:00 - [   DEBUG] - No block fetched. Waiting before next fetch.
2024-07-05T08:03:49.605712000+00:00 - [    INFO] - Stopping prefetcher...
2024-07-05T08:03:49.606550000+00:00 - [   DEBUG] - Prefetch task cancelled.
2024-07-05T08:03:50.165107000+00:00 - [   DEBUG] - Prefetching blocks stopped.
2024-07-05T08:03:50.166289000+00:00 - [   DEBUG] - Executor shutdown complete.
2024-07-05T08:03:50.167117580+00:00 - [   DEBUG] - RS Fetcher - Stopping...
2024-07-05T08:03:50.167417734+00:00 - [   DEBUG] - RS Fetcher - 1 Consumer workers started
2024-07-05T08:03:50.167651211+00:00 - [   DEBUG] - RS Fetcher - BitcoinClient-1 worker exited.
2024-07-05T08:03:50.167956917+00:00 - [   DEBUG] - RS Fetcher - Orderer-0 worker exited.
2024-07-05T08:03:50.168178239+00:00 - [   DEBUG] - RS Fetcher - BitcoinClient-0 worker exited.
2024-07-05T08:03:50.168531672+00:00 - [   DEBUG] - RS Fetcher - Fetcher-0 worker exited.
2024-07-05T08:03:50.168799742+00:00 - [   DEBUG] - RS Fetcher - Fetcher-1 worker exited.
2024-07-05T08:03:50.169070809+00:00 - [   DEBUG] - RS Fetcher - Extractor-0 worker exited.
2024-07-05T08:03:50.169279026+00:00 - [   DEBUG] - RS Fetcher - Reporter-0 worker exited.
```

File: counterparty-core/counterpartycore/lib/backend/rsfetcher.py [hunk 3]
```diff
@@ -83,7 +83,7 @@ def get_prefetched_block(self, height):
         try:
             logger.debug(f"Looking for Block {height} in prefetch queue...")
             while height not in self.prefetch_queue:
-                if self.prefetch_queue_size == 0:
+                if self.prefetch_queue_size == 0 and self.prefetch_queue_initalized:
```
Comment (adamkrellenstein) [inline counterparty-core/counterpartycore/lib/backend/rsfetcher.py:86]: So we don't get warnings on startup.

File: counterparty-rs/src/indexer/logging.rs (hunk 1 above)
Comment (ouziel-slama) [reply counterparty-rs/src/indexer/logging.rs]: done

**Pull request merged**